code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1053,19 +1053,19 @@
 
 	add_traits(list(TRAIT_NO_DNA_SCRAMBLE, TRAIT_BADDNA, TRAIT_BORN_MONKEY), SPECIES_TRAIT)
 
-/mob/living/carbon/human/proc/is_atmos_sealed(additional_flags = null, check_hands = FALSE, ignore_chest_pressureprot = FALSE)
+/mob/living/carbon/human/proc/is_atmos_sealed(additional_flags = null, check_hands = FALSE, alt_flags = FALSE)
 	var/chest_covered = FALSE
 	var/head_covered = FALSE
 	var/hands_covered = FALSE
 	for (var/obj/item/clothing/equipped in get_equipped_items())
 		// We don't really have space-proof gloves, so even if we're checking them we ignore the flags
 		if ((equipped.body_parts_covered & HANDS) && num_hands >= default_num_hands)
 			hands_covered = TRUE
-		if (!isnull(additional_flags) && !(equipped.clothing_flags & additional_flags))
+		if (!alt_flags && !isnull(additional_flags) && !(equipped.clothing_flags & additional_flags))
```

Comment:
Shouldn't it be:
```
		if (alt_flags && !isnull(additional_flags) && !(equipped.clothing_flags & additional_flags))
```?

Reply:
alt_flags param makes it test for *either* pressureprot or additional_flags, you'll be reversing its behavior here. No, this is correct